overlord/hookstate/ctlcmd/set.go

```diff
@@ -20,18 +20,28 @@
 package ctlcmd
 
 import (
+	"encoding/json"
 	"fmt"
+	"strings"
 
 	"github.com/snapcore/snapd/i18n"
 )
 
 type setCommand struct {
 	baseCommand
+
+	Positional struct {
+		ConfValues []string `positional-arg-name:"<conf value>" description:"configuration value (key=value)" required:"1"`
```

Comment:
s/&lt;conf value&gt;/key=value/

and we can drop the description I think. This is clear from context.